CustomKeystore/pom.xml

```diff
@@ -33,25 +33,18 @@ under the License.
         <kda.version>2.0.0</kda.version>
         <kda.runtime.version>1.2.0</kda.runtime.version>
         <java.version>11</java.version>
-        <flink.version>1.13.2</flink.version>
+        <flink.version>1.15.2</flink.version>
         <scala.binary.version>2.12</scala.binary.version>
         <maven.compiler.source>${java.version}</maven.compiler.source>
         <maven.compiler.target>${java.version}</maven.compiler.target>
     </properties>
 
     <dependencies>
-        <!-- KDA dependencies -->
```

Comment:
Why are we deleting this dependency and adding it again below but with an older version?